### ISSUE TITLE
Distinguish server so counts on different servers are seperated

### ISSUE THREAD

**Issue by ddddkim0525** [ddddkim0525/art-bot#2]
I want to use this bot in other daily submission challenges I do (I'm currently doing a blender one on the side). So the data should also store the guild name and only add up submission counts on that specific guild rather than just using author.

**Comment by zoticusbarns** [comment]
I believe this feature can be implemented by:
1. Modifying the database: adding a new column "guild" in the "submission" table. With "general" as the default value so the data in the current database can be kept.
2. Either have separate channels for different challenges, or add a new user input parameter in the !submit command
3. Same as point 2, the weekly submission summary function will also need to be modified.

**Comment by ddddkim0525** [comment]
Yup I'll try to work on this feature, as it seems managable for me
